keps/sig-apps/1847-autoremove-statefulset-pvcs/README.md

```diff
@@ -305,9 +310,29 @@ In order to update the PVC ownerReference, the `buildControllerRoles` will be up
 
 ### Test Plan
 
-1. Unit tests
+[X] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+#### Unit tests
+
+In `pkg/controller/statefulset/`,
+
+- `TestStatefulPodControlUpdatePodClaimForRetentionPolicy`
+- `TestStatefulPodControlRetainDeletionPolicyUpdate`
+- `TestStatefulPodControlRetentionPolicyUpdate`
+- `TestStatefulPodControlRetentionPolicyUpdateMissingClaims`
+- `TestOrphanedPodsWithPVCDeletePolicy`
+- `TestStaleOwnerRefOnScaleup`
+- `stateful_set_control_test`
+   - Added `runTestOverPVCRetentionPolicies` to augment existing tests with
+  coverage for retention policies.
+   - `checkClaimInvarients` extended to test new feature.
+  
+##### e2e/integration tests
+
+Tests added to `test/e2e/apps/statefulset.go`.
```

Comment:
Same as above, although here:
```
- <test>: <link to test coverage>
```

Reply:
Also e2e and integration should be two distinct paragraphs.

Reply:
Done.

I wasn't aware of the integration tests, as noted these need to be added. Or should I just record the plan so that the KEP doesn't have to be updated when I add these (as I plan to do in the next month)?